python/pyarrow/tests/test_csv.py

```diff
@@ -1321,8 +1321,17 @@ def test_cancellation(self):
         raise_signal = util.get_raise_signal()
 
         # Make the interruptible workload large enough to not finish
-        # before the interrupt comes, even in release mode on fast machines
-        large_csv = b"a,b,c\n" + b"1,2,3\n" * 200_000_000
+        # before the interrupt comes, even in release mode on fast machines.
+        last_duration = 0.0
+        workload_size = 100_000
+
+        while last_duration < 1.0:
+            print("workload size:", workload_size)
```

Comment:
Looks pytest hides the output by default. Curious to see the workload size big enough for M1.

Reply:
You can pass the `-s` option to pytest. Alternatively, if the test fails, the output is shown.